### DISCUSSION TITLE
Gauge background

### DISCUSSION THREAD

**Discussion by Welsyntoffie** [frangoteam/FUXA#1106] [General]
When I add a gauge the background is white. 
Where do I change the background of a gauge to transparent?

**Comment by robbudge** [comment]
Same as any other object.
Click the object then click the transparent color in the bottom bar. White Square with red x

**Reply by Welsyntoffie** [reply]
Jeeees, took me while. Thank you. I used the right click interactivity properties.
Never even noticed the color bar at the main bottom screen is for color changing.

**Reply by robbudge** [reply]
I did exactly the same, just could not see the wood for the trees
